### PR TITLE
adding 10242022 release notes

### PR DESCRIPTION
**Pull request by olsenme**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-10-24
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is retiring `v1.22.x` on December 4th 2022. Please upgrade your clusters to `v1.23` and above.
```
Comment (allyford) [inline CHANGELOG.md:9]: I'd recommend adding a link here to upgrade documentation: https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+* Docker is no longer supported as a container runtime on Windows. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* Kubernetes 1.25 is in preview. We support Kubernetes 1.25.2.
+
+### Release notes
+
+* Features
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
+
+* Behavior Changes
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
```
Comment (allyford) [inline CHANGELOG.md:22]: This is a pretty large change for customers since we're now enforcing this policy. We may want to move it to announcements. I'm also working on sending out comms for this as well

Comment (allyford) [reply CHANGELOG.md:22]: Can we change the announcement to: 
AKS will be enforcing the [de-allocated clusters](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters) policy which specifies that manually de-allocating clusters renders the cluster out of support. Starting November 1, 2022 clusters with zero nodes will be stopped after 30 days.

Comment (allyford) [reply CHANGELOG.md:22]: Internal subs will be shutdown after 1d - thoughts on if that should be included in release notes?

Comment (kaarthis) [reply CHANGELOG.md:22]: I dont see it as a problem given we dont have another place to put 'internal only' announcements.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+* AKS is retiring `v1.22.x` on December 4th 2022. Please upgrade your clusters to `v1.23` and above.
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. `beta.kubernetes.io/arch=` and `beta.kubernetes.io/os=`  are still applied by kubelet in kubernetes code.
+* Docker is no longer supported as a container runtime on Windows. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* Kubernetes 1.25 is in preview. We support Kubernetes 1.25.2.
+
+### Release notes
+
+* Features
```
Comment (helayoty) [inline CHANGELOG.md:18]: May we add the new supported regions for virtual-node?

Comment (olsenme) [reply CHANGELOG.md:18]: What are these?

Comment (helayoty) [reply CHANGELOG.md:18]: https://github.com/virtual-kubelet/azure-aci/releases/tag/v1.4.6

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
+
+* Behavior Changes
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Provided the option to remove the LoadBalancer service tunnelfront port 9000 for clusters using Konnectivity.
```
Comment (allyford) [inline CHANGELOG.md:24]: Line 24 should be removed - @alvinli222

File: CHANGELOG.md (hunk 2 above)
Comment (shashankbarsin) [inline CHANGELOG.md:22]:
```suggestion
  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/azure/aks/support-policies#stopped-or-de-allocated-clusters)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+### Release notes
+
+* Features
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
+
+* Behavior Changes
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Provided the option to remove the LoadBalancer service tunnelfront port 9000 for clusters using Konnectivity. 
+  * Fixed a bug to allow cluster creation where both AAD and local accounts are disabled.
```
Comment (shashankbarsin) [inline CHANGELOG.md:25]: isn't is 'disallow'? the bug was that we were allowing clusters before where both AAD and local account could have been disabled

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,37 @@
...
+
+* Features
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
+
+* Behavior Changes
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Provided the option to remove the LoadBalancer service tunnelfront port 9000 for clusters using Konnectivity. 
+  * Fixed a bug to allow cluster creation where both AAD and local accounts are disabled. 
+  * Fixed a bug in the 10/16 overlaymgr release where azure-cns does not mount /var/run/azure-cns on the host filesystem. When azure-cns pods are restarted, they lose track of which IPs have been assigned; if these IPs are re-assigned, pod networking breaks. This will happen in Cilium-enabled clusters whenever azure-cns crashes or overlaymgr deploys a new version of the azure-cns daemonset.
```
Comment (shashankbarsin) [inline CHANGELOG.md:26]: overlaymgr will not be relevant for customer facing release notes. Can we review this message and update to parts that are only relevant to end customers?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,37 @@
...
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Provided the option to remove the LoadBalancer service tunnelfront port 9000 for clusters using Konnectivity. 
+  * Fixed a bug to allow cluster creation where both AAD and local accounts are disabled. 
+  * Fixed a bug in the 10/16 overlaymgr release where azure-cns does not mount /var/run/azure-cns on the host filesystem. When azure-cns pods are restarted, they lose track of which IPs have been assigned; if these IPs are re-assigned, pod networking breaks. This will happen in Cilium-enabled clusters whenever azure-cns crashes or overlaymgr deploys a new version of the azure-cns daemonset. 
+  * Fixed bug where when a cluster is updated, it triggers a reconcile cluster operation which will remove the setting  aks-vnet -> subnet -> service endpoints  which is set by csi driver when provisioning volume using NFS protocol.
+  * Fixed a bug which prevented executing a command in a container in an svc cluster.
```
Comment (shashankbarsin) [inline CHANGELOG.md:28]: svc cluster? is this relevant to end customers?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,37 @@
...
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Provided the option to remove the LoadBalancer service tunnelfront port 9000 for clusters using Konnectivity. 
+  * Fixed a bug to allow cluster creation where both AAD and local accounts are disabled. 
+  * Fixed a bug in the 10/16 overlaymgr release where azure-cns does not mount /var/run/azure-cns on the host filesystem. When azure-cns pods are restarted, they lose track of which IPs have been assigned; if these IPs are re-assigned, pod networking breaks. This will happen in Cilium-enabled clusters whenever azure-cns crashes or overlaymgr deploys a new version of the azure-cns daemonset. 
+  * Fixed bug where when a cluster is updated, it triggers a reconcile cluster operation which will remove the setting  aks-vnet -> subnet -> service endpoints  which is set by csi driver when provisioning volume using NFS protocol.
+  * Fixed a bug which prevented executing a command in a container in an svc cluster. 
+* Component Updates
+  * Updated workload identity image to v.1.14.0.
```
Comment (shashankbarsin) [inline CHANGELOG.md:30]: do we have link to its release notes?

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,36 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-10-24
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is retiring `v1.22.x` on December 4th 2022. Please [upgrade](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster?tabs=azure-cli) your clusters to `v1.23` and above.
```
Comment (shashankbarsin) [inline CHANGELOG.md:9]:
```suggestion
* AKS is retiring `v1.22.x` on December 4th 2022. Please [upgrade](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) your clusters to `v1.23` and above.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,36 @@
...
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. `beta.kubernetes.io/arch=` and `beta.kubernetes.io/os=`  are still applied by kubelet in kubernetes code.
+* Docker is no longer supported as a container runtime on Windows. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* Kubernetes 1.25 is in preview. We support Kubernetes 1.25.2.
+* AKS will be enforcing the de-allocated clusters [policy](https://learn.microsoft.com/azure/aks/support-policies#stopped-or-de-allocated-clusters) which specifies that manually de-allocating clusters renders the cluster out of support. Starting November 1, 2022 clusters with zero nodes will be stopped after 30 days. Internal subs will be shutdown after 1d.
```
Comment (shashankbarsin) [inline CHANGELOG.md:15]: 'Internal subs' is not relevant for end customers. Can we handle comms for this separately internally?

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,36 @@
...
+* Virtual Node is supported in [these] (https://github.com/virtual-kubelet/azure-aci/commit/a33f6278694ecf8f71b6c182f6b31a90fd5d60cc) additional regions. 
+
+### Release notes
+
+* Features
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: Is there a specific list of v5 SKUs? or is it all?

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,36 @@
...
+
+* Behavior Changes
+  * The cpu limits for cloud-node-manager, csi drivers, and kube-proxy have been removed.
+  * Fixed a bug to disallow cluster creation where both AAD and local accounts are disabled.  
+  * Fixed bug where when a cluster is updated, it triggers a reconcile cluster operation which will remove the setting  aks-vnet -> subnet -> service endpoints  which is set by csi driver when provisioning volume using NFS protocol.
+
+* Component Updates
+  * Updated [workload identity image](https://github.com/Azure/azure-workload-identity/releases/tag/v0.14.0) to v.1.14.0.
```
Comment (shashankbarsin) [inline CHANGELOG.md:29]: link to tag says "v0.14.0" but the text says "v.1.14.0"

File: CHANGELOG.md (hunk 10 above)
Comment (olsenme) [reply CHANGELOG.md:15]: @allyford

File: CHANGELOG.md (hunk 11 above)
Comment (olsenme) [reply CHANGELOG.md:21]: I am not sure on this one. Asked Patrick. His doc says this sku is not supported

File: CHANGELOG.md (hunk 10 above)
Comment (allyford) [reply CHANGELOG.md:15]: I'd say remove the internal subs part - I'll send it in comms

**Review by miwithro — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]
